clustering/infinispan/extension/src/main/java/org/jboss/as/clustering/infinispan/subsystem/StateTransferResourceDefinition.java

```diff
@@ -53,7 +53,7 @@
     static final PathElement PATH = pathElement("state-transfer");
 
     enum Attribute implements org.jboss.as.clustering.controller.Attribute {
-        CHUNK_SIZE("chunk-size", ModelType.INT, new ModelNode(10000)),
+        CHUNK_SIZE("chunk-size", ModelType.INT, new ModelNode(512)),
```

Comment:
Fixed now. Thanks for noticing the XSD because it turned out the description was incorrect and could have been very misleading.